lib/vagrant-seil/command.rb

```diff
@@ -0,0 +1,64 @@
+require "optparse"
+require "vagrant"
+
+module VagrantPlugins
+  module Seil
+    class Command < Vagrant.plugin(2, :command)
+      def self.synopsis
+        "save configuration of running SEIL into specified host file"
+      end
+
+      def execute
+        options = {}
+
+        opts = OptionParser.new do |o|
+          o.banner = "Usage: vagrant seil-save-to [options] [name]"
```

Comment:
IMHO, the command should make clear difference from following, which only saves to VM's temporal storage. So I added apparent keyword as "-host".

```
% vagrant ssh default -- save-to flashrom
```

But I agree that it's too lengthy. It's your call on which to go.
Either way, banner in line 15 (or command itself) is not what it should be. I'll fix it.